generators/reference_quality/tiktoken_token_counter/code_snippet_refinery.md

```diff
@@ -0,0 +1,11 @@
+```python
+import tiktoken 
+
+ATTRIBUTE: str = "text" # only text attributes
+ENCODING_NAME: str = "cl100k_base"
```

Comment:
i'm not familiar with tiktoken to much. Is this the best default option?

Reply:
Yes, according to OpenAI, this is the encoder used for GPT-3.5 and GPT-4. https://cookbook.openai.com/examples/how_to_count_tokens_with_tiktoken

It would also be possible to do it like this: 
`encoding = tiktoken.encoding_for_model("gpt-3.5-turbo")`
Which results in the same, but might be more easily understandable?

Reply:
both have pros and cons so i'm fine with this. Maybe add a comment after with "open AI encoder for gpt"

Reply:
@FelixKirschKern As this brick is not listed on the Notion page for cognition, I would argue we should move it to "text_analytics" group in the generator section. What do you think?

Reply:
yes sounds good